### ISSUE TITLE
Siege tank attack "turret shake" animation sometimes doesn't play

### ISSUE THREAD

**Issue by Joshua-Leibold** [Ahli/sc2xml#131]
https://i.gyazo.com/b06d631fcb98cfd193aa9b55261e2a27.mp4

Don't quote me on it but I think it only happens when the Siege tank is not facing the target

**Comment by Ahli** [comment]
The turret shake animation is the attack animation. I managed to somehow break it as well. The Tank did not play the attack animation for a few shots until the weapon stopped firing.

The tank has 2 events to start the attack's animation. The tank has 2 weapons.
I guess that the fake weapon stops the animation in some scenario. Maybe you need to move sideways between shots and go back to attacking again before the real weapon's cooldown ends.

I assume it is fixable via hardcoding the attack animation bracket to the real weapon instead of `*` (any).

**Comment by Ahli** [comment]
### Fix
With this fix, the attack animation is always playing.

```xml
    <CActorUnit id="SiegeTank">
        <On index="65" Terms="WeaponStart.90mmCannons.AttackStart"/>
        <On index="66" Terms="WeaponStart.90mmCannons.AttackStart; AnimPlaying Birth"/>
        <On index="67" Terms="WeaponStop.90mmCannons.AttackStop"/>
        <On index="72" removed="1"/>
    </CActorUnit>
```
![image](https://github.com/Ahli/sc2xml/assets/5763784/110ead70-e8f7-4b8e-af51-425629d7d7c8)

[test map](https://www.dropbox.com/scl/fi/rzyvi967ncby589i2iaho/siegeTankAttackAnimNotPlayingFix.SC2Map?rlkey=d16mnz5d58bo6c9sxbuz9n74u&dl=1)

**Comment by Joshua-Leibold** [comment]
This was fixed in 5.0.13
